crates/bevy_render/src/camera/active_cameras.rs

```diff
@@ -5,12 +5,60 @@ use bevy_ecs::{
 };
 use bevy_utils::HashMap;
 
+/// A reference to the entity holding the camera to use for a given content type.
+///
+/// This acts as a link between:
+/// - an [`ExtractedCamera`], that is, a camera found to be valid for the current frame as
+///   analyzed during the [`RenderStage::Extract`] stage, and indirectly the underlying
+///   [`Camera`] it was extracted from;
+/// - a set of render phases used to render a specific kind of content (2D or 3D).
+///
+/// A set of [`ActiveCamera`] for each supported kind of content is pre-built at startup and
+/// added to the [`ActiveCameras`] resource. Their `entity` field is then updated each frame
+/// to point to the entity holding the [`Camera`] to use for that content. See [`ActiveCameras`]
+/// for details.
+///
+/// Note that the name "active camera" is a misnomer, and does not mean "a Camera which is active",
+/// as multiple [`Camera`] components can be added to the same world, and the [`Camera`] component
+/// itself has no concept of being "active" or not. Instead, it refers to the camera being the one
+/// selected for rendering the content type defined by `name`.
+///
+/// [`ExtractedCamera`]: super::ExtractedCamera
```

Comment:
Does this style of link render properly in cargo doc? I always use the `[NAME](PATH)` style, but I'm not convinced it's superior.

Reply:
I tend to prefer that one because it keeps a relative similarity in size between source and rendered content, which makes it easier to read the comment in code (as opposed to rendered). This also avoids long "unbreakable" lines containing a long link. And yes, as far as I tried this renders correctly. This is an official `rustdoc` link type : https://doc.rust-lang.org/rustdoc/linking-to-items-by-name.html

Reply:
Awesome, SGTM.